.github/workflows/ci.yml

```diff
@@ -459,7 +459,7 @@ jobs:
       shell: bash
       run: |
         mkdir ${GITHUB_WORKSPACE}/wheels
-        docker run --rm -v ${GITHUB_WORKSPACE}/wheels:/data_swap_guest kratosmultiphysics/kratos-wheelbuilder-linux master 2
+        docker run --rm -v ${GITHUB_WORKSPACE}/wheels:/data_swap_guest kratosmultiphysics/kratos-wheelbuilder-linux master 4
```

Comment:
the runners only have 2 CPUs
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources